mkdocs/config/config_options.py

```diff
@@ -308,6 +308,7 @@ def post_validation(self, config, key_name):
         # ensure a well-formed edit_uri
         if edit_uri:
             if not edit_uri.startswith(('?', '#')) \
+                    and config['repo_url'] is not '' \
```

Comment:
This change fixes the problem we experienced, I modified the above conditionals as well since they were unreachable before.